decidim-proposals/spec/shared/proposal_form_examples.rb

```diff
@@ -139,7 +139,6 @@
     context "when the has address checkbox is checked" do
       let(:has_address) { true }
 
-      # rubocop:disable RSpec/EmptyExampleGroup
```

Comment:
This comments should not be deleted. Since this PR will be about decidim-debates, we should not change things in other components.